dynamiq/nodes/retrievers/chroma.py

```diff
@@ -55,6 +55,13 @@ def __init__(self, **kwargs):
     def vector_store_cls(self):
         return ChromaVectorStore
 
+    @property
+    def vector_store_params(self):
```

Comment:
It's already present in parent `VectorStoreNode`. Do that params have some changes comparing to parent?

Reply:
Yes, it`s specified to ignore content_key